qr-code-auth/README.md

```diff
@@ -0,0 +1,286 @@
+# Collection - QrCodeAuth
+
+## API Endpoints
+
+|                                       Route                                        |                                         Description                                          |
+| :--------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------: |
+|                      [POST /qr-code-auth](#post-qr-code-auth)                      |                            Creates a new authentication document.                            |
+|                       [GET /qr-code-auth](#get-qr-code-auth)                       |           Retrieves all the authentication document belonging to a specified user            |
+| [PATCH /qr-code-auth](#patch-qr-code-authuser_iduser_idis_authorizedis_authorized) | Updates the is_authorized field of an existing qr-code-auth document for the specified user. |
+
+## POST /qr-code-auth
+
+    Creates a new qr-code-auth document.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **device_info** (required, string): Specifies the device information associated with the authentication authentication.
+    - **is_authorized** (required, boolean): Indicates whether the authentication document is authorized or not.
+    - **access_token** (required, string): Specifies the access token associated with the authentication document.
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  - rds-session: `<JWT>`
```

Comment:
Will we be having an RDS-session cookie before sending the POST request, @shreya-mishra ?

Reply:
updated it so that RDS-session cookie is no more needed